java/opendcs/src/main/java/decodes/rledit/RefListFrame.java

```diff
@@ -104,18 +103,39 @@ public class RefListFrame extends JFrame
     private UnitConverterDb deletedConverter = null;
     private String []deletedDte = null;
 
-    private SeasonListTableModel seasonListTableModel = new SeasonListTableModel();
-    private SortingListTable seasonsTable = new SortingListTable(seasonListTableModel,
-        SeasonListTableModel.colWidths);
+    private SeasonListTableModel seasonListTableModel;
+    private SortingListTable seasonsTable;
 
     private final OpenDcsDatabase database;
 
     /**
      * constructor for RefListFrame
      */
-    public RefListFrame(OpenDcsDatabase database)
+    public RefListFrame(OpenDcsDatabase database) throws DecodesException
     {
         this.database = database;
+        log.info("RefListFrame:initDecodes()");
+		Optional<Database> db = this.database.getLegacyDatabase(Database.class);
+		if (db.isPresent()) 
+        {
```

Comment:
This should only be done in a main function. not in individual classes, even the GUI ones.